src/views/AnalysisView.vue

```diff
@@ -177,7 +177,7 @@ function updateScaling(min, max){
       <image-download-menu
         :image-name="image.basename"
         :fits-url="image.url || image.fits_url"
-        :jpg-url="analysisStore.imageUrl"
```

Comment:
This is the generated scaled jpeg the view was using, not the original large jpeg